### PR TITLE
[N6] Use AddressSearch for identity form

### PR DESCRIPTION
**Pull request by robertjchen**
### Details

Thanks for the review! Here we're simply replacing the address fields on the personal info step with a new `AddressSearch` component.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/180829

### Tests / QA Steps
1. Head on over to new.expensify.com.dev or https://staging.new.expensify.com/
2. Make and Validate a new account
3. The Create Menu should pop up at the lower left, but if it doesn't click the green plus button down there
4. Click 'New Workspace' (then click 'Expensify Card' if you're on mobile)
5. Click 'Get Started' in the modal that pops up
6. Click 'Connect Manually' in the VBA modal that comes up on the right hand side
7. Use the following info for the first step
    a) Routing number 011401533
    b) Account number 1111222233331111
8. Click 'Save & Continue'
9. Type 42 in the 'Company Address' field
10. Verify that the address suggestions appear.
11. Click one of them and verify that it enters that data into the 'Company Address' field and that you are able to continue to the next step

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
Screen Shot 2021-10-13 at 9 43 25 AM

#### Mobile Web
Screen Shot 2021-10-13 at 10 44 40 AM

#### Desktop
Screen Shot 2021-10-13 at 7 44 03 AM

#### iOS
Screen Shot 2021-10-13 at 7 54 36 AM

#### Android
Screen Shot 2021-10-13 at 10 25 59 AM

### CODE REVIEW

File: src/pages/ReimbursementAccount/RequestorStep.js [hunk 1]
```diff
@@ -92,6 +92,11 @@ class RequestorStep extends React.Component {
             city: 'requestorAddressCity',
             state: 'requestorAddressState',
             zipCode: 'requestorAddressZipCode',
+
```
Comment (NikkiWines) [inline src/pages/ReimbursementAccount/RequestorStep.js:95]: NAB: extra line

**Review by NikkiWines — DISMISSED** [review]
Looks good and works well.

Screen Shot 2021-10-13 at 2 22 42 PM

Minor comment that if the form doesn't submit successfully it looks like only the `Company address` field gets cleared:

https://user-images.githubusercontent.com/16747822/137214696-89457636-cb75-4f3e-b984-48858c6ca7ac.mov

**Comment by robertjchen** [comment]
Updooted and retested 👍

**Review by NikkiWines — APPROVED** [review]
Tested and works well!! 🎉

**Pull request merged**